src/commcare_cloud/manage_commcare_cloud/manage_commcare_cloud.py

```diff
@@ -7,14 +7,15 @@
 from commcare_cloud.manage_commcare_cloud.get import Get
 from commcare_cloud.manage_commcare_cloud.list_vault_keys import ListVaultKeys
 from commcare_cloud.manage_commcare_cloud.install import Install
-from commcare_cloud.manage_commcare_cloud.make_docs import MakeDocs
+from commcare_cloud.manage_commcare_cloud.make_docs import MakeDocs, MakeChangelog
 
 COMMAND_TYPES = [
     Install,
     Get,
     ListVaultKeys,
     Configure,
     MakeDocs,
+    MakeChangelog
```

Comment:
nit: a trailing comma here will make diffs cleaner next time. No need to update this now, just noting that as a nice diff-friendly practice.

Reply:
https://github.com/dimagi/commcare-cloud/pull/1979/commits/93abc0c36d3ee7f0bf594951438519f8d795e00a